src/index.html

```diff
@@ -1,14 +1,37 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css" >
+
   </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+  <div class="countainer">
+    <header class="header">
+      <a class="logo" href="logo">
+        <img src="../src/images/logo.png" alt="logo" >
+      </a>
+      <nav class="categories">
+        <ul class="categories__ul">
+          <li class="categories__li"><a href="#" class="is-active">Apple</a></li>
+          <li class="categories__li"><a href="#">Samsung</a></li>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>
          <li class="nav__item"><a href="#"  class="nav__link">Samsung</a></li>
```
suggest more meaningful naming of classes.style link by class.